src/status_im/contexts/wallet/wallet_connect/modals/common/footer/view.cljs

```diff
@@ -45,6 +45,7 @@
        [standard-authentication/slide-button
         {:size                :size-48
          :track-text          slide-button-text
+         :keycard-supported?  true
```

Comment:
`:on-complete         (when sign-on-keycard? #(on-auth-success ""))`

Reply:
@flexsurfer what's the difference between the `:on-complete` and `:on-auth-success` props here? Looking at the one used in send (`status-im.contexts.wallet.send.transaction-confirmation.view`) and seems like the changes are recent:

```clj
  [standard-auth/slide-button {...
                                          :on-complete (when sign-on-keycard?
                                                         #(rf/dispatch
                                                           [:wallet/prepare-signatures-for-transactions
                                                            :send
                                                            ""]))
                                          :on-auth-success
                                          (fn [psw]
                                            (rf/dispatch
                                             [:wallet/prepare-signatures-for-transactions :send
                                              psw]))}]
```

Reply:
`on-complete` is used when you need to sign on keycard, on-auth-success used if you need password (for keycard also, it will work)

Reply:
it confusing yes, when we want to sign on keycard, we just need a signal from the slide button that it was activated, we don't need auth process